abb_irb1200_support/urdf/irb1200_5_90_macro.xacro

```diff
@@ -104,42 +145,48 @@
       <axis xyz="0 0 1"/>
       <parent link="${prefix}base_link"/>
       <child link="${prefix}link_1"/>
-      <limit effort="0" lower="-2.967" upper="2.967" velocity="5.027"/>
+      <limit effort="1000" lower="-2.967" upper="2.967" velocity="5.027"/>
```

Comment:
If we don't know the effort limits, we usually keep them at `0`. Is that not possible?

Reply:
It seems that when using `effort_controllers` the max effort specified in the URDF is used to limit the joint effort. Making it `0`, does not seem to mean that there is no limit, rather, it means that the joint cannot do any effort.

Trying to trace a bit the code to justify this I found [where the limits are loaded](https://github.com/ros-simulation/gazebo_ros_pkgs/blob/2901fe6cc16c86a68cb01d664a45276cac9ec712/gazebo_ros_control/src/default_robot_hw_sim.cpp#L393) and that the limits [are enforced](https://github.com/ros-simulation/gazebo_ros_pkgs/blob/2901fe6cc16c86a68cb01d664a45276cac9ec712/gazebo_ros_control/src/default_robot_hw_sim.cpp#L302-L303) when 'writing' to the sim, which results on [all joint commands being 0](https://github.com/ros-controls/ros_control/blob/f38bf6d7c21e744e37b254d86f1ab7796ea803f7/joint_limits_interface/include/joint_limits_interface/joint_limits_interface.h#L309).

So, to the best of my knowledge, we cannot leave `effort=0` if we want to use effort controllers (instead of position controllers as used [in other packages](https://github.com/ros-industrial/abb_experimental/blob/ce65133979980bab79a4f8f8c3afe78c4eb126c7/abb_irb120_gazebo/config/irb120_3_58_arm_controller.yaml#L2)).